DATA/SOLAR/loadouts_wrecks.ini

```diff
@@ -476,11 +476,11 @@ nickname = SECRET_c_co_elite_bw02b
 equip = fc_c_gun_pulse_light01, HpWeapon01
 equip = fc_c_gun_pulse_light01, HpWeapon02
 equip = fc_c_turret_neutron_heavy01, HpTurret01
-equip = cv_shield_graviton_medium_adv01, HpShield01
+equip = li_shield_graviton_medium_adv01, HpShield01

```

Comment:
Liberty Shield on an RH fighter?

Reply:
That is a Corsair fighter, not a Rheinland one. In this set of fixes I changed the loadouts to not use the deprecated shields, rather than fixing all switchups in the loadouts yet. It's noted for my next set of fixes for the upcoming patch, however.

Reply:
Right. I let myself be confused by how Github displayed the changes and cut off a whole section here. Making it look like an RH Fighter by nickname

Reply:
but even then, liberty is wrong. must be a positron shield in this case